http/codegen/service_data.go

```diff
@@ -1105,11 +1139,24 @@ func buildResultData(svc *service.Data, s *httpdesign.ServiceExpr, e *httpdesign
 				}
 				notag = i
 			}
+			// Project response body if body type is a result type and if the
+			// endpoint is not a result type with multiple views
```

Comment:
... otherwise the response body is used to initialize a viewed result type that is then validated and transformed into a result type (??)